Documentation/Contributing/Feature_Contribution_Process.md

```diff
@@ -1,10 +1,11 @@
-10/1/2019: This page is deprecated because it provides guidelines for contributing very large systems to MRTK before the 2.0 release. After the 2.0 release, large changes need to be performed more carefully, and the process for this is not yet decided. We expect most MRTK contributions to have much smaller changes than what is covered here.
-
 # Feature Contribution Process
 
+> [!WARNING]
```

Comment:
Perhaps it is time to refresh / remove this article as it is out of date, and as this comment states, large changes need to be handled carefully.

Reply:
I did a quick pass on refreshing this article. I'm not against replacing it with a new doc in an upcoming iteration